platzky/admin/admin.py

```
@@ -3,31 +3,28 @@
 from flask import Blueprint, render_template, session
 
 
-def create_admin_blueprint(login_methods, db, locale_func):
+def create_admin_blueprint(login_methods, cms_modules):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)